tests/python/contrib/test_hexagon/infrastructure.py

```diff
@@ -18,36 +18,43 @@
 """ Hexagon testing infrastructure """
 
 import tvm
+from tvm import te
 import numpy
 
 
 def ceildiv(o, d):
     return tvm.tir.floordiv(o + d - 1, d)
 
 
-def get_packed_activation_layout(shape_nhwc, block_shape, packed_C=True):
+def get_block_shape():
+    return 8, 8, 32
+
+
+def get_filter_block_shape():
+    return 8, 32, 4
```

Comment:
It's not obvious why these particular numbers are chosen. Should there be comments explaining the reasons for using them?

Reply:
Added a basic comment: "defines inner filter block shape: 8i32o41"